Lesson3/Task8.java

```diff
@@ -0,0 +1,14 @@
+public class Task8 {
+	public static void main(String[] args) {
+		String str = "1234 5678 9012 3456";
+		
+		//string to char array
+		char[] chars = str.toCharArray();
+		for (int i =0; i < chars.length-4; i++) {
+		if (chars[i] != ' ')
+		chars[i] = '*';
+		}
+		for(int j=1; j < chars.length; j++)
+		System.out.print(chars[j]);
```

Comment:
This one is good as well but try to use method which were demonstrated on the lesson. Write bellow one more way with using substring and replace.